lib/traject/macros/csv.rb

```diff
@@ -11,5 +11,11 @@ def column(header_or_index, options = {})
         accumulator.concat(result) unless result.first.to_s.empty?
       end
     end
+
+    def numismatic_id
+      lambda do |row, accumulator, _context|
+        accumulator << row['RecordId'].tr('.', '_')
```

Comment:
I believe it's about keep easily routable URL parameters (with the dots, we've have to go in and change the Rails default patterns.. which is fine and doable, but probably not worth it for this project).

Maybe it'd help if we called this method `normalize_numismatic_id`, and used, say, `row['RecordId'].to_param` to capture our intent a little better?

I also wonder if we want to use our institution-prefixed ids here (and anywhere else we're not)?